ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/FSharpParensUtil.fs

```diff
@@ -186,3 +186,8 @@ let addParens (expr: IFSharpExpression) =
 let addParensIfNeeded (expr: IFSharpExpression) =
     if not (needsParens expr expr) then expr else
     addParens expr
+
+
+let tryGetParentParens expr =
+    let paren = ParenExprNavigator.GetByInnerExpression(expr)
```

Comment:
Does it need to work with nested parens? `((x -> x))`
Can [IgnoreParentParens](https://github.com/JetBrains/fsharp-support/blob/dbd044ece175a19f9581a5fe1a64b068d50c492f/ReSharper.FSharp/src/FSharp.Psi/src/Impl/FSharpImplUtil.cs#L684) be used instead?

Reply:
@auduchinok, yes indeed